distribution/project-manager/THIRD-PARTY/NOTICE

```diff
@@ -256,6 +256,126 @@ The license file can be found at `licenses/Bouncy_Castle_Licence.txt`.
 Copyright notices related to this dependency can be found in the directory `org.bouncycastle.bcutil-jdk15on-1.70`.
 
 
+'org-netbeans-api-annotations-common', licensed under the The Apache Software License, Version 2.0, is distributed with the project-manager.
```

Comment:
I understand we need this module in `engine` runtime. However this is a report for `project-manager`. Do we need it here?

Moreover project manager is compiled using `native-image` and this kind of sampling only works in classical HotSpot JVM mode.

Reply:
I put the sampler in the utils library, and probably it was used by the project manager. Maybe it's a good idea to create a separate project for it.